src/App.js

```diff
@@ -28,22 +29,11 @@ const alex = {
 
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
+    <Person {...misha} />
```

Comment:
Will be better to define some field in props of Person component and pass here person object
```suggestion
    <Person person={misha} />
```